packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -126,18 +159,52 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
     });
   }
 
+  _onInputTriggerTypingEventsHandler(e: InputEvent): void {
+    if (this._isValuableTextEntered()) {
```

Comment:
So, if we have "       1" text and we remove last character, onTypingStart will not be called? Looks strange to me

Reply:
[30f954d8b2d174191c45c26c9ef35cce6e9da515]